components/campaigns/CampaignsFilter.tsx

```diff
@@ -0,0 +1,150 @@
+"use client";
+import { Input } from "@/components/ui/input";
+import {
+  Select,
+  SelectContent,
+  SelectItem,
+  SelectTrigger,
+  SelectValue,
+} from "@/components/ui/select";
+import { Calendar, CheckIcon, Search, Server } from "lucide-react";
+import { useState } from "react";
+import DatePicker from "../DatePicker";
+const availableMailboxes = [
+  "john@mycompany.com",
+  "sarah@mycompany.com",
+  "mike@mycompany.com",
+  "lisa@mycompany.com",
+  "david@mycompany.com",
+];
```

Comment:
Move to a mock like 

```
const domains = {
'mycompany.com': [
  "john@mycompany.com",
  "sarah@mycompany.com",
  "mike@mycompany.com",
  "lisa@mycompany.com",
  "david@mycompany.com",
]
}
```

Or a similar structure

Reply:
Chek if you can reuse the data from [`lib/data/campaigns.ts`](lib/data/campaigns.ts)